tslint.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
NIT: this file could use tabs like tsconfig.json

Reply:
Fixed :)